### ISSUE TITLE
[packages] zfs-testing repository for EL8

### ISSUE THREAD

**Issue by philfry** [openzfs/zfs#10188]
Hi,

as I'm running the most recent kernel from elrepo-kernel on centos 8 the zfs 0.8.3 kernel module is no longer compiling for me. I wanted to try more recent builds, thus enabling the zfs-testing repositories.

Unfortunately, the only response I get from
http://download.zfsonlinux.org/epel-testing/8.1/x86_64/repodata/repomd.xml
is a 403 forbidden (also tested with `8`, `8.0` and `8.2`).

Kind regards

**Comment by behlendorf** [comment]
We haven't yet provided an epel-testing repository for 8.x, which is why you're seeing that 403 error.  Could you post your kernel version and the failure you're seeing?

**Comment by philfry** [comment]
Hi Brian, thanks for your reply.

Using kernel 5.6.2 I get
```
/var/lib/dkms/zfs/0.8.3/build/include/spl/sys/time.h:88:15: error: unknown type name ‘time_t’
/var/lib/dkms/zfs/0.8.3/build/include/spl/sys/time.h:108:18: error: storage size of ‘ts’ isn’t known
```
and stuff, which is probably already fixed by 2c3a83701dd185cadb30db4556256534e2930c7d.
I wanted to try out a "nightly" zfs-dkms but I guess I can use the sources as well or wait for the next stable release. Was just wondering why accessing the testing-repository failed.

**Comment by behlendorf** [comment]
A zfs-testing repository was added for CentOS/RHEL 8.2.  It contains the latest OpenZFS 2.0 release candidate.
